tools/qontract_cli.py

```diff
@@ -3912,5 +3920,47 @@ def get_input(
     print(resource.json(exclude={"data": {FLAG_RESOURCE_MANAGED_BY_ERV2}}))
 
 
+@external_resources.command()
+@click.argument("provision-provider", required=True)
+@click.argument("provisioner", required=True)
+@click.argument("provider", required=True)
+@click.argument("identifier", required=True)
+@click.pass_context
+def request_reconciliation(
+    ctx, provision_provider: str, provisioner: str, provider: str, identifier: str
+):
+    """Marks a resource as it needs to get reconciled. The itegration will reconcile the resource at
+    its next iteration.
```

Comment:
what is the use case of manual status change?

Reply:
It's useful for testing new module versions, debugging purposes, etc. It's not intended to be used extensively but it helps in determined cases. Specially now with the initial migration.